### PR TITLE
Math Magician Testing

### PR DESCRIPTION
**Pull request by Irirwanirira**
## Our Pr Consists of : 

- [x] We created snapshots for our function
- [x] We used Jest for testing
- [x] We tested the operate and calculate function.

### Thank you for reviewing our project.

### CODE REVIEW

File: src/logic/testLogic/operate.test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  test('operate function should return 3', () => {
+    expect(Number(operate(obj.num4, obj.num2, '-'))).toBe(3);
+  });
+  test('operate function should return 12', () => {
+    expect(Number(operate(obj.num1, obj.num2, 'x'))).toBe(12);
+  });
+  test('operate function should return 3', () => {
+    expect(Number(operate(obj.num2, obj.num3, '÷'))).toBe(2);
+  });
+});
```
Comment (Munsa1) [inline src/logic/testLogic/operate.test.js]: - [x] Great job testing some of the functionality of the operations, kindly note that you have to test all of the functionality of the operations of the calculator not just the four mathematical operands. Kindly do so
- [x] Also do not forget to test for the modulus `%` operator

File: src/__Function_test__/__snapshots__/Calculators.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,2107 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`it  crashing 1`] = `
+Object {
+  "asFragment": [Function],
+  "baseElement": <body>
+    <div>
+      <div
```
Comment (Munsa1) [inline src/__Function_test__/__snapshots__/Calculators.test.js.snap:8]: - [x] There should be the usage of Jest snapshots for testing all the components. You can add `--coverage` to Identify which lines have not been covered. Kindly add more snapshot testing
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Irirwanirira @AbayomiOlaoye , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._
- [x] Kindly make your readme file more descriptive by adding all collaborators information
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @Irirwanirira and @AbayomiOlaoye, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**